src/components/Nav.js

```diff
@@ -1,37 +1,69 @@
 import React, { useContext } from 'react';
 import { Link } from 'react-router-dom';
+import { Icon } from '@iconify/react';
+import mapIcon from '@iconify-icons/fa-regular/map';
+import unorderedListOutlined from '@iconify-icons/ant-design/unordered-list-outlined';
+import bxCurrentLocation from '@iconify-icons/bx/bx-current-location';
+import settingsLine from '@iconify-icons/clarity/settings-line';
+import helpIcon from '@iconify-icons/carbon/help';
+
 import { LocationsContext } from '../context/LocationsContext';
 import LoadingMessage from '../components/LoadingMessage';
-import './Nav.css';
+import './Nav.scss';
```

Comment:
Does this file need to be SASS? I don't see anything on first glance that requires it 🤔